scripts/CSSRE/lib/py/exceptions.py

```diff
@@ -8,12 +8,12 @@ def __init__(self, msg):
         super().__init__(msg)
 
 
-class StatefulsetExecConnectionError(Exception):
+class ServiceExecConnectionError(Exception):
     def __init__(self, msg):
-        details = "This can happen if there are no pods in the statefulset in a ready state or a pod restarted during the process. Ensure there are ready pods and try again."
+        details = "This can happen if the bootstrap server for the service does not exist/cannot be reached."
```

Comment:
The error message about this potentially failing if there aren't pods in a ready state is still valid here. 
In fact I think it's the more important part to mention (and more likely cause of a connection error) is pods crashing. When testing the script I've encountered connection errors because a pod restarts so it's good to note that once a pods i running you can just rerun the script

Reply:
Hey Rob, thank you for testing this. No problem, I've changed the logger name and updated the error message to give some more info on the pods. 
The process for rolling out changes to production is [this](https://source.redhat.com/groups/public/openshiftplatformsre/wiki/backplane_scripts#how-to-promote-to-production), right? Would you like me to start that as soon as this gets merged?